hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/fees/CongestionPricingTest.java

```diff
@@ -65,10 +62,8 @@ public class CongestionPricingSuite {
             requirement = {PROPERTY_OVERRIDES, THROTTLE_OVERRIDES},
             overrides = {"contracts.maxGasPerSec", "fees.percentCongestionMultipliers", "fees.minCongestionPeriod"})
     final Stream<DynamicTest> canUpdateMultipliersDynamically() {
-        var artificialLimits = protoDefsFromResource("testSystemFiles/artificial-limits-congestion.json");
-        var defaultThrottles = protoDefsFromResource("testSystemFiles/throttles-dev.json");
```

Comment:
was this file (probably no longer used?) supposed to be `throttles-def{ault}` not `throttles-dev`?

Reply:
It really shouldn't have ever existed; we should always have been taking a snapshot of file `0.0.123` contents and then restoring those like in this PR.

Deleted now!